### PR TITLE
Fix Parametric test return point

### PR DESCRIPTION
**Pull request by Affie**

### CODE REVIEW

File: test/testParametric.jl [hunk 1]
```diff
@@ -28,21 +29,28 @@ for i in 0:3
...
 # @test isapprox(vardict[:x0].val, [10, 10, -pi], atol = 1e-3)
 # @test isapprox(vardict[:x4].val, [10, 10, -pi], atol = 1e-3)
-
-@test isapprox(vardict[:x0].val, [10, 10, -pi], atol = 1e-3) || isapprox(vardict[:x0].val, [10, 10, pi], atol = 1e-3)
-@test isapprox(vardict[:x1].val, [0, 10, -pi/2], atol = 1e-3)
-@test isapprox(vardict[:x2].val, [0, 0, 0], atol = 1e-3)
-@test isapprox(vardict[:x3].val, [10, 0, pi/2], atol = 1e-3)
-@test isapprox(vardict[:x4].val, [10, 10, -pi], atol = 1e-3) || isapprox(vardict[:x4].val, [10, 10, pi], atol = 1e-3)
+ϵ = getPointIdentity(M)
+@test isapprox(M, vardict[:x0].val, exp(M, ϵ, hat(M,ϵ,[10, 10, -pi])), atol = 1e-3)
```
Comment (dehann) [inline test/testParametric.jl:49]:
```suggestion
@test isapprox(M, vardict[:x0].val, exp_lie(M, hat(M,ϵ,[10, 10, -pi])), atol = 1e-3)
```

Comment (dehann) [reply test/testParametric.jl:49]: Ronny suggested this on SO(3) stuff I was doing somewhere

Comment (Affie) [reply test/testParametric.jl:49]: It defaults to ProductRepr and will give an error. Plus its the wrong exp.

Comment (dehann) [reply test/testParametric.jl:49]: ah, okay

**Comment by Affie** [comment]
CI Failure looks like an unrelated intermittent numerical issue.
This will only pass with IIF master.

**Pull request merged**